editors/sc-ide/core/main.cpp

```diff
@@ -124,6 +127,22 @@ int main( int argc, char *argv[] )
     if (startInterpreter)
         main->scProcess()->startLanguage();
 
+    // setup HelpBrowser server
+    QWebSocketServer server("SCIDE HelpBrowser Server", QWebSocketServer::NonSecureMode);
+    if (!server.listen(QHostAddress::LocalHost, 12345)) {
```

Comment:
should we have another port number or is this going to be fine?

Reply:
A question I have as well!

Reply:
pick something in the range of 12323 and 12344 for instance.
based on: 
https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.txt
12345 should be fine though: LocalHost is apparently hard-coded anyway and 12345 is not in the well-known ports.
Only an overzealous security tool (e.g. on Windows) might be irritated since there are trojans out-there using 12345. (https://www.speedguide.net/port.php?port=12345)